sql/mozfun/norm/get_windows_info/README.md

```diff
@@ -0,0 +1,49 @@
+# Windows Names, Versions, and Builds
+
+### Summary
+This function is primarily designed to parse the field `os_version` in table `mozdata.default_browser_agent.default_browser`. Though untested, the function should also parse any generic Microsoft Windows OS version strings.
```

Comment:
might suggest changing any to most, and dating when this was written, e.g.

"most generic Microsoft Windows OS version strings as of December 2022"

I realize this is mentioned later, but being explicit here might help